ci/00_title.md

```diff
@@ -86,27 +70,61 @@ plugins: RevealMarkdown, RevealChalkboard, RevealHighlight, RevealMath.KaTeX, Re
 
 </style>
 
-# Continuous _What_ Now...?
+# Continuous Integration (CI)
 
-<div class="blockquote">
-  Automating the integration of code changes from multiple contributors into a single software project.
-<h4>&mdash; <a href="atlassian.com/continuous-delivery/continuous-integration">Atlassian</a></h4>
+<div class="blockquote-wrapper fragment">
+  <div class="blockquote">
+     It is the practice of integrating changes to a project made by individuals into a main, shared version frequently.
```

Comment:
How about this?

```suggestion
     means keeping the main (shared) version of a project up to date via frequent or automated integration of changes.
```

Reply:
I like it more! Although we cannot quote the turing way on this.